### PR TITLE
Sync JA sidenav with EN

### PR DESCRIPTION
**Pull request by kbatuigas**
# Description
Add more translations for reorganized sidenav in JA
*Does not yet include Server 4.1
*Execution environments section has not been reorganized yet

NOTE: Pages that have not yet been translated to JA are commented out in the side nav (to serve as placeholder / easier to remember where they should go). Screenshots below have not been updated to reflect this.

### Screenshots: 
image
image
image
image
image
image
image
image
image
image
image
image
image
image

# Reasons
This unblocks the work to be done to split the sidenav YAML file into two separate files for EN and JA.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Comment by kbatuigas** [comment]
> @kbatuigas I noticed titles are being repeated. Is this the expected behavior? or is it just cut off? Screenshot 2023-03-28 at 11 43 11 AM
> 
> Screenshot 2023-03-28 at 11 42 56 AM
> 
> Screenshot 2023-03-28 at 11 42 44 AM

Yes, it's just where I cut the screenshots off, to "mark" where I had stopped

**Review by rosieyohannan — APPROVED** [review]
LGTM AFAICT! 🎉

File: jekyll/_data/sidenav.yml [hunk 1]
```diff
@@ -1481,162 +1557,207 @@ ja:
...
+            link: ja/browser-testing
+            i18n_name: ブラウザーテスト
+          - name: Code coverage
+            link: ja/code-coverage
+            i18n_name: コードカバレッジ
+          - name: Fail fast (Open preview)
+            link: ja/fail-fast
+            i18n_name: フェイルファスト
+      - name: Monitor tests
+        i18n_name: モニターテスト
```
Comment (makotom) [inline jekyll/_data/sidenav.yml:1676]:
```suggestion
        i18n_name: テストの管理
```

File: jekyll/_data/sidenav.yml [hunk 2]
```diff
@@ -1387,88 +1418,133 @@ ja:
...
             i18n_name: 変更のEC2へのデプロイ
-          - name: Notify a Slack channel of a paused workflow
-            link: ja/notify-a-slack-channel-of-a-paused-workflow
-            i18n_name: 停止したワークフローのSlackチャンネルへの通知
+      - name: Reference
+        i18n_name: リファレンス
+        children:
+          - name: Orbs concepts
+            link: ja/orb-concepts
+            i18n_name: Orbコンセプト
```
Comment (makotom) [inline jekyll/_data/sidenav.yml:1426]:
```suggestion
            i18n_name: Orb のコンセプト
```

File: jekyll/_data/sidenav.yml [hunk 3]
```diff
@@ -1272,84 +1284,99 @@ ja:
...
+          - name: Insights glossary
+            i18n_name: インサイトの用語集
+            link: ja/insights-glossary
+          - name: Insights API
+            i18n_name: インサイトAPI
+            link: https://circleci.com/docs/api/v2#tag/Insights
 
-  - name: Orbs
+  - name: Package and re-use config with orbs
     i18n_name: Orb
```
Comment (makotom) [inline jekyll/_data/sidenav.yml:1370]:
```suggestion
    i18n_name: Orb を使った再利用可能な設定のパッケージ化
```

**Review by makotom — COMMENTED** [review]
I've added 3 proposals - @yuki-sakamoto-san @kbatuigas could you check them?

**Comment by kbatuigas** [comment]
> I've added 3 proposals - @yuki-sakamoto-san @kbatuigas could you check them?

Thank you @makotom , looks good to me!

**Pull request merged**